src/LIBMONGOC_VERSION_CURRENT

```diff
@@ -1 +1 @@
-1.17.0-beta
+1.18.0-20200519+gitc63f92c305
```

Comment:
Switching to 1.18-dev is necessary as features still need to be back ported into 1.17 from master. Since we'll require 1.17 when this is released, I'm not changing the requirement in config.m4 for the time being.